src/reports/components/ReportsLayout.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Outlet } from 'react-router-dom';
+import PageLayout from '../../common/components/PageLayout';
+import ReportsMenu from './ReportsMenu';
+
+const ReportsLayout = () => (
+  <PageLayout
+    menu={<ReportsMenu />}
+    breadcrumbs={['reportTitle', 'reportCombined']}
```

Comment:
The breadcrumbs is different per report.

Reply:
It's true, I hadn't noticed it.
I made these changes to fix it.

- 17ade9707e14bd04e071a59a610cd34c4e3780a2
- 8651b8dc0d29cb9c0709c1092eb01e6e2126332a